### PR TITLE
TL Localization - Chapter 6, section 4 & 5 (1,515 words)

### PR DESCRIPTION
**Pull request by heywens**

### CODE REVIEW

File: book/06-github/sections/4-managing-organization.asc [hunk 1]
```diff
@@ -1,72 +1,72 @@
...
+Bukod sa mga single-user account, ang GitHub ay may tinatawag na mga Organisasyon.
+Tulad ng mga personal na account, ang mga account ng Organisasyon ay may namespace kung saan umiiral ang lahat ng kanilang mga proyekto, ngunit maraming iba pang mga bagay ay naiiba.
+Ang mga account na ito ay kumakatawan sa isang grupo ng mga tao na may ibinahaging pagmamay-ari ng mga proyekto, at mayroong maraming mga kasangkapan upang pamahalaan ang mga subgroup ng mga tao.
+Karaniwan ang mga account na ito ay ginagamit para sa mga grupo ng Open Source (tulad ng ``perl'' o ``rails'') o mga kompanya (tulad ng ``google'' o ``twitter'').
 
-==== Organization Basics
+==== Mga Pangunahing Kaalaman sa Organisasyon
 
-An organization is pretty easy to create; just click on the ``+'' icon at the top-right of any GitHub page, and select ``New organization'' from the menu.
+Ang isang organisasyon ay medyo madali malikha; i-click lamang ang icon na ``+'' na nasa itaas na kanan ng anumang pahina ng GitHub, at piliin ang ``Bagong Organisasyon'' mula sa menu.
```
Comment (josteinfrancois) [inline book/06-github/sections/4-managing-organization.asc]: organisasyon

File: book/06-github/sections/4-managing-organization.asc [hunk 2]
```diff
@@ -1,72 +1,72 @@
...
+Ang organisasyon ay nauugnay sa mga indibidwal na tao sa pamamaraan ng koponan, na kung saan ay isang grupo lamang ng mga indibidwal na account ng gumagamit at mga repositoryo sa loob ng organisasyon at kung anong uri ng pag-access mayroon ang mga taong iyon sa mga repositoryo.
 
-For example, say your company has three repositories: `frontend`, `backend`, and `deployscripts`.
-You'd want your HTML/CSS/JavaScript developers to have access to `frontend` and maybe `backend`, and your Operations people to have access to `backend` and `deployscripts`.
-Teams make this easy, without having to manage the collaborators for every individual repository.
+Halimbawa, sabihin ang iyong kumpanya ay may tatlong repositoryo: `frontend`, `backend`, at `deployscripts`.
+Gugustuhin mo na ang iyong mga developer ng HTML/CSS/JavaScript na magkaroon ng access sa `frontend` at marahin sa `backend`, at ang iyong mga tao sa Operasyon na magkaroon ng access sa `backend` at `deployscripts`.
```
Comment (josteinfrancois) [inline book/06-github/sections/4-managing-organization.asc]: at marahil sa `backend`

File: book/06-github/sections/4-managing-organization.asc [hunk 3]
```diff
@@ -1,72 +1,72 @@
...
+.Ang pahina ng Organisasyon.
 image::images/orgs-01-page.png[]
 
-To manage your Teams, you can click on the Teams sidebar on the right hand side of the page in <<_org_page>>.
-This will bring you to a page you can use to add members to the team, add repositories to the team or manage the settings and access control levels for the team.
-Each team can have read only, read/write or administrative access to the repositories.
-You can change that level by clicking the ``Settings'' button in <<_team_page>>.
+Upang mapamahalaan ang iyong mga Koponan, maaari mong i-click ang sidebar na Koponan sa kanang bahagi ng pahina sa <<_org_page>>.
+Ito ay magdadala sa iyo sa isang pahina na maaari mong magamit upang magdagdag ng mga miyembro sa koponan, magdagdag ng mga repositoryo sa koponan o pamahalaan ang settings at kontrol sa mga antas ng pag-access para sa koponan.
+Ang bawat koponan ay maaari magkaroon ng pagbasa lamang, pagbasa/pagsulat o access na administratibo sa mga repositoryo.
```
Comment (josteinfrancois) [inline book/06-github/sections/4-managing-organization.asc]: maaaring magkaroon

File: book/06-github/sections/4-managing-organization.asc [hunk 4]
```diff
@@ -1,72 +1,72 @@
...
+Maaari mo baguhin ang antas na iyon sa pamamagitan ng pag-click sa pindutan na ``Settings'' sa <<_team_page>>.
```
Comment (josteinfrancois) [inline book/06-github/sections/4-managing-organization.asc]: Maaari mong

File: book/06-github/sections/4-managing-organization.asc [hunk 5]
```diff
@@ -1,72 +1,72 @@
...
-.The Team page.
+.Ang pahina ng Koponan.
 image::images/orgs-02-teams.png[]
 
-When you invite someone to a team, they will get an email letting them know they've been invited.
+Kapag inaanyayahan mo ang isang tao sa isang koponan, makakakuha sila ng email na nagpapaalam sa kanila na sila ay inanyayahan.
 
-Additionally, team `@mentions` (such as `@acmecorp/frontend`) work much the same as they do with individual users, except that *all* members of the team are then subscribed to the thread.
-This is useful if you want the attention from someone on a team, but you don't know exactly who to ask.
+Bukod pa rito, ang `@mentios` ng koponan (tulad ng `@acmecorp/frontend`) ay gumagana nang kapareho katulad ng ginagawa nila sa mga indibidwal na gumagamit, maliban na ang *lahat* ng mga miyembro ng koponan ay naka-subscribe sa thread.
```
Comment (josteinfrancois) [inline book/06-github/sections/4-managing-organization.asc]: `@mentions`

File: book/06-github/sections/5-scripting.asc [hunk 6]
```diff
@@ -1,63 +1,63 @@
...
-.Services and Hooks configuration section.
-image::images/scripting-01-services.png[Services and hooks]
+.Seksyon ng pagsasaayos sa mga Serbisyo at mga Hook.
+image::images/scripting-01-services.png[Mga serbisyo at mga hook]
 
-There are dozens of services you can choose from, most of them integrations into other commercial and open source systems.
-Most of them are for Continuous Integration services, bug and issue trackers, chat room systems and documentation systems.
-We'll walk through setting up a very simple one, the Email hook.
-If you choose ``email'' from the ``Add Service'' dropdown, you'll get a configuration screen like <<_service_config>>.
+Mayroong dose-dosenang mga serbisyo ang maaari mong pagpipilian, karamihan sa mga ito ay mga pagsasama sa iba pang mga komersyal at mga sistema na open ssource.
```
Comment (josteinfrancois) [inline book/06-github/sections/5-scripting.asc]: open source

**Review by josteinfrancois — COMMENTED** [review]
Hello @heywens . I have reviewed your work. Minor changes only.

**Comment by heywens** [comment]
Hey @josteinfrancois, thanks for taking time on reviewing my contribution. All necessary changes have been applied. Thanks!

**Pull request merged**